src/components/TarsiaGrid/index.jsx

```diff
@@ -22,9 +23,9 @@ const TarsiaGrid = ({ id, grid, values }) => {
 
   return (
     <g id={id}>
-      {grid && grid.map((location, index) => {
+      {hexGrid && hexGrid.map((location, index) => {
         const valueArray = formatGridLocationValueArray({ values, locationValues: location.values })
-        return <Triangle key={index} row={location.row} col={location.col} values={valueArray} config={appConfig.triangle} />
+        return <Triangle key={index} row={location.row} col={location.col} values={valueArray} config={config.triangle} />
```

Comment:
I'm purposely still using a config prop here. It's easy to go overboard with using context and, seeing as this is the only place `Triangle` is called, the values it receives are custom and that it is only one component deep, I might as well pass it down to keep `Triangle` dumb (stateless)